src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700;900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a
+        href="#MOYO"
+        class="img"
+      >
+        <img
+          src="images\logo.png"
```

Comment:
The file path for the image source uses a backslash ('\\') which is typically used in Windows file paths. For web URLs, it's recommended to use a forward slash ('/') to ensure compatibility across different environments.